File: src/main/Java/src/Vendedor.java
class Vendedor extends Colaborador{
    private double comissao;
    private final double salarioMinimo = 1320.00;
    private long numeroDeVendas;

    public Vendedor(String nome, long cpf, double comissao, long numeroDeVendas) {
        super(nome, cpf);
        this.comissao = comissao;
        this.numeroDeVendas = numeroDeVendas;
    }

    @Override
    public double CalcularSalario() {
        return this.salarioMinimo + (this.salarioMinimo * this.comissao);
    }
}

File: src/main/kotlin/Vendedor.kt
import Colaborador as Colaborador1

class Vendedor(nome: String, cpf: Long, comissao: Double, ) : Colaborador1(nome, cpf) {
    val comissao = comissao
    val salarioMinimo = 1320.00

    override fun CalcularSalario(): Double {
        return salarioMinimo + (this.salarioMinimo + this.comissao)
    }

}